pg/prod.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM postgres:14.3-alpine as dataloader
+
+
+COPY scripts/init.sql /docker-entrypoint-initdb.d/
+ENV POSTGRES_USER=postgres
+ENV POSTGRES_PASSWORD=change-me
+ENV PGDATA=/data
+
+COPY enzymes-deploy.zip  /input_data/enzymes.zip
```

Comment:
in this dockerfile, the `input_data` folder is not copied, only the zipped file (produced by `proj.sh`)